src/CMakeLists.txt

```diff
@@ -140,6 +140,8 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     string(APPEND LEAN_EXTRA_LINKER_FLAGS " -Wl,--stack,${LEAN_WIN_STACK_SIZE}")
     set(EXTRA_UTIL_LIBS ${EXTRA_UTIL_LIBS} -lpsapi)
   endif()
+  # Link against winsock2
+  string(APPEND LEAN_EXTRA_LINKER_FLAGS " -lws2_32")
```

Comment:
During #5135 we linked against all of `psapi,iphlpapi,userenv,ws2_32,dbghelp,ole32` in `prepare_llvm_mingw.sh` (IIRC this was the result of running `pkg-config --libs libuv` on a MinGW test machine). Are you sure we don't need the other ones here?